### PR TITLE
Update version, changelog and release notes for beta20 release

### PR DESCRIPTION
**Pull request by alerickson**
# PR Summary
Update changelog, release notes and module version for 3.0.20-beta20 release.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 3.0.20-beta20
+
+- Move off of NuGet client APIs and use direct REST API calls  (#1023)
```
Comment (anamnavi) [inline CHANGELOG.md]:
```suggestion
- Move off of NuGet client APIs and use direct REST API calls for remote repositories (#1023)

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 3.0.20-beta20
+
+- Move off of NuGet client APIs and use direct REST API calls  (#1023)
+
+### Bug Fixes
```
Comment (anamnavi) [inline CHANGELOG.md:8]: I believe @SydneyhSmith checked and closed issues that got resolved with this release. She may know if others should be listed.

**Review by anamnavi — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (SydneyhSmith) [reply CHANGELOG.md:8]: https://github.com/PowerShell/PowerShellGet/issues/697
https://github.com/PowerShell/PowerShellGet/issues/754
https://github.com/PowerShell/PowerShellGet/issues/755

are the issues that I was able to close

File: src/PowerShellGet.psd1 [hunk 3]
```diff
@@ -45,7 +45,18 @@
...
+## 3.0.20-beta20
+
+- Move off of NuGet client APIs and use direct REST API calls (#1023)
+            
+### Bug Fixes
+- Updates to dependency installation (#1010) (#996) (#907)
+- Update to retrieving all packages installed on machine (#999)
+- PSResourceInfo version correctly displays 2 or 3 digit version numbers (#697)
+- Using `Find-PSresource` with `-CommandName` or `-DSCResourceName` parameters returns an object with a properly expanded ParentResource member (#754)
+- `Find-PSResource` no longer returns duplicate results (#755)
```
Comment (anamnavi) [inline src/PowerShellGet.psd1:57]:
```suggestion
- `Find-PSResource` no longer returns duplicate results (#755)
- `Find-PSResource` lists repository 'PSGalleryScripts' which does not exist for `Get-PSResourceRepository` (#1028)
```

**Review by SydneyhSmith — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
+- Move off of NuGet client APIs and use direct REST API calls for remote repositories (#1023)
+
+
+### Bug Fixes
+- Updates to dependency installation (#1010) (#996) (#907)
+- Update to retrieving all packages installed on machine (#999)
+- PSResourceInfo version correctly displays 2 or 3 digit version numbers (#697)
+- Using `Find-PSresource` with `-CommandName` or `-DSCResourceName` parameters returns an object with a properly expanded ParentResource member (#754)
+- `Find-PSResource` no longer returns duplicate results (#755)
```
Comment (anamnavi) [inline CHANGELOG.md:13]:
```suggestion
- `Find-PSResource` no longer returns duplicate results (#755)
- `Find-PSResource` lists repository 'PSGalleryScripts' which does not exist for `Get-PSResourceRepository` (#1028)
```

**Pull request merged**